pkg/operator/ceph/cluster/cluster_test.go

```diff
@@ -362,6 +367,11 @@ func TestTelemetry(t *testing.T) {
 			telemetry.DeviceSetNonPortableKey:    "0",
 			telemetry.NetworkProviderKey:         "",
 			telemetry.ExternalModeEnabledKey:     "true",
+			telemetry.K8sNodeCount:               "3",
+			telemetry.CephFSNodeCount:            "0",
+			telemetry.RBDNodeCount:               "0",
+			telemetry.NFSNodeCount:               "0",
+			telemetry.CephNodeCount:              "-1",
```

Comment:
How about a unit test that checks where the nodes are not 0 or -1?

Reply:
Added